email_alert_service/presenters/unpublishing_message_presenter.rb

```diff
@@ -0,0 +1,63 @@
+class UnpublishingMessagePresenter
+  EMAIL_DATE_FORMAT = "%l:%M%P, %-d %B %Y".freeze
+
+  def initialize(unpublishing_scenario, document, page_title)
+    @unpublishing_scenario = unpublishing_scenario
+    @document = document
+    @page_title = page_title
+  end
+
+  def call
+    [
+      ["Body:", page_title_markdown],
+      ["Change made:", unpublishing_scenario_note],
+      ["Time updated:", formatted_time],
+    ].join("\n\n")
+  end
```

Comment:
I think this is wrong, and you don't actually want the page title here.

The content doc has:

```
Subject:
Update from GOV.​UK for: [page title]

Body:
[Page title](/link)

Page summary:
[Page summary]

Change made:
This page was removed from GOV.UK. It’s been replaced by: [URL of redirect]

^You’ve been automatically unsubscribed from this page because it was removed.
```

Here, the "Subject:" and "Body:" indicate the relevant bits of the email, they're not part of the actual text.

So, the subject is

```
Update from GOV.​UK for: [page title]
```

And the body is

```
[Page title](/link)

Page summary:
[Page summary]

Change made:
This page was removed from GOV.UK. It’s been replaced by: [URL of redirect]


^You’ve been automatically unsubscribed from this page because it was removed.
```

email-alert-api fills in the subject for us based on the list title, so that's fine.  It also fills in this body for us:

```
Update from GOV.UK for:

# #{list.title}

---


#{middle_section}

---

[Change your email preferences](#{manage_url})
```

Where the `middle_section` is the markdown submitted to the API endpoint.  So, since we have decided to drop the page summary for now, this method should return something of the form:

```
Change made:
This page was removed from GOV.UK. It’s been replaced by: [URL of redirect]

^You’ve been automatically unsubscribed from this page because it was removed.
```

Reply:
Do you think we need the `^You’ve been automatically unsubscribed from this page because it was removed.`
Given there is already content under [Change made: ](https://github.com/alphagov/email-alert-service/pull/481/files#diff-f9f00044fff88e629db2ba700b16ec435dabaf4490b321d4ca3596a94f38b8d3R29-R33)?

Reply:
The content doc has that message, so I think we should keep it.